test/addons/rbd-mirror/start

```diff
@@ -268,4 +276,7 @@ wait_until_pool_mirroring_is_healthy(cluster2)
 deploy_vrc_sample(cluster1)
 deploy_vrc_sample(cluster2)
 
+deploy_vgrc_sample(cluster1)
+deploy_vgrc_sample(cluster2)
```

Comment:
This looks fine, but it would be little bit nicer to group all the resources in a kustomization directory and apply all of them with one command, if you want to try.